test/devices_tests/climate_test.py

```diff
@@ -1258,6 +1270,33 @@ async def test_process_heat_cool(self):
         climate_mode.process(telegram)
         assert climate_mode.controller_mode == HVACControllerMode.HEAT
 
+    async def test_process_humidity(self):
+        """Test process / reading telegrams from telegram queue. Test if humidity is processed correctly."""
+        xknx = XKNX()
+        climate = Climate(xknx, "TestClimate", group_address_humidity="1/2/16")
+
+        telegram = Telegram(
+            destination_address=GroupAddress("1/2/16"),
+            payload=GroupValueWrite(DPTHumidity().to_knx(45.6)),
+        )
+        climate.process(telegram)
+        assert climate.humidity.value == 45.6
+
+    async def test_process_callback_humidity(self):
+        """Test process / reading telegrams from telegram queue. Test if callback is executed when receiving humidity."""
+
+        xknx = XKNX()
+        climate = Climate(xknx, "TestClimate", group_address_humidity="1/2/16")
+        after_update_callback = Mock()
+        climate.register_device_updated_cb(after_update_callback)
+
+        telegram = Telegram(
+            destination_address=GroupAddress("1/2/16"),
+            payload=GroupValueWrite(DPTHumidity().to_knx(45.6)),
```

Comment:
```suggestion
            payload=GroupValueWrite(DPTHumidity.to_knx(45.6)),
```
No need for an instance.